Detectors/TRD/workflow/io/src/TRDDigitWriterSpec.cxx

```diff
@@ -75,8 +75,9 @@ o2::framework::DataProcessorSpec getTRDDigitWriterSpec(bool mctruth, bool writeT
                                 // setting a custom callback for closing the writer
                                 MakeRootTreeWriterSpec::CustomClose(finishWriting),
                                 BranchDefinition<std::vector<o2::trd::Digit>>{InputSpec{"input", "TRD", "DIGITS"}, "TRDDigit"},
-                                BranchDefinition<std::vector<o2::trd::TriggerRecord>>{InputSpec{"trinput", "TRD", "TRGRDIG"}, "TriggerRecord", (writeTrigRec ? 1 : 0)},
-                                std::move(labelsdef))();
+                                BranchDefinition<std::vector<o2::trd::TriggerRecord>>{InputSpec{"tracklettrigs", "TRD", "TRKTRGRD"}, "TrackTrg"})();
```

Comment:
This would break the simulation again. Couldn't you send two trigger records from the raw reader? Both `TRGRDIG` and `TRKTRGRD` types?

Reply:
I know ;-) have not figured that one out yet, there is a rawtreewriter, but gave up on that.

Reply:
reverted to orignal one so as not to break simulation again